### PR TITLE
Fixed an abnormal behavior about the default config file

### PR DESCRIPTION
**Pull request by tivizi**
what's the abnormal behavior?

i stored my `config.yaml` to `$HOME/.config/neofs-cli`, but can't load to `neofs-cli`

### CODE REVIEW

**Comment by alexvanin** [comment]
Please update your commit message according to the commit style in this repository. It should contain PR number in the header and sign-off, e.g.

```
[#626] Support default config file

Signed-off-by: Name Surname <email>
```

File: cmd/neofs-cli/modules/root.go [hunk 1]
```diff
@@ -123,9 +124,11 @@ func initConfig() {
 			os.Exit(1)
 		}
 
-		// Search config in home directory with name ".main" (without extension).
-		viper.AddConfigPath(home)
-		viper.SetConfigName(".config/neofs-cli")
+		// Search config in `$HOME/.config/neofs-cli/` or `$PWD` with name "config.yaml"
+		viper.AddConfigPath(filepath.Join(home, ".config", "neofs-cli"))
+		viper.AddConfigPath(".")
```
Comment (alexvanin) [inline cmd/neofs-cli/modules/root.go:129]: I am not sure it is good idea to look for `config.yaml` file in `$PWD` by default. We can accidentally use config of other application that can also have `rpc` of `wallet` section. This won't happen if it is located in separate `.config/neofs-cli` dir. Let's stick only with that.

**Comment by alexvanin** [comment]
Also check DCO / Sign-Of linter [error](https://github.com/nspcc-dev/neofs-node/pull/626/checks?check_run_id=2906307364).

**Comment by tivizi** [comment]
I changed my name in GitHub : )

**Review by alexvanin — APPROVED** [review]

**Pull request merged**